### PR TITLE
updated docs to remove reference to here() and updated gitignore

### PR DESCRIPTION
**Pull request by nmaves**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,9 @@ coverage
 dist
 .opt-in
 .opt-out
+
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+.idea/*
```
Comment (kentcdodds) [inline .gitignore]: This should be in your global gitignore file: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Do you mind removing it from this PR?

**Comment by kentcdodds** [comment]
This looks perfect. Just that `.gitignore` thing and we're good 👍

**Comment by nmaves** [comment]
Damn you are fast.  Now go answer my question in your latest video on udemy :)  Then I might have another idea for this project.

**Comment by kentcdodds** [comment]
Thanks!

> Now go answer my question in your latest video on udemy

I'm not aware that I have any videos on udemy! Link?

**Comment by nmaves** [comment]
My bad egghead.io :)

https://egghead.io/forums/lesson-discussion/topics/intro-to-the-production-webpack-course?page=1

**Comment by kentcdodds** [comment]
Somehow I missed that! I normally see all the messages that come through. Thanks for pinging me.

**Comment by nmaves** [comment]
This should close #2

**Pull request merged**